src/gui/wizard/owncloudwizard.cpp

```diff
@@ -152,6 +173,16 @@ int OwncloudWizard::calculateLongestSideOfWizardPages(const QList<QSize> &pageSi
     });
 }
 
+void OwncloudWizard::reject()
+{
+    QWizard::reject();
+
+    // Quit if the user hasn't completed the wizard
```

Comment:
nitpick: I'd say "has never completed the wizard". It's fine to not complete it if you did it at least once in the past.